public/images/common/check.svg

```diff
@@ -0,0 +1,4 @@
+<svg width="24" height="24" viewBox="0 0 24 24" fill="none" xmlns="http://www.w3.org/2000/svg">
+<path fill-rule="evenodd" clip-rule="evenodd" d="M8.99901 19C8.73301 19 8.47901 18.895 8.29201 18.707L2.29301 12.704C1.90201 12.313 1.90201 11.68 2.29301 11.29C2.68401 10.899 3.31601 10.899 3.70701 11.29L8.99901 16.586L20.291 5.29301C20.682 4.90201 21.314 4.90201 21.705 5.29301C22.096 5.68401 22.096 6.31601 21.705 6.70701L9.70601 18.707C9.51901 18.895 9.26501 19 8.99901 19Z" fill="currentColor"/>
+<path d="M8.99901 19V19C8.73301 19 8.47901 18.895 8.29201 18.707L2.29301 12.704C1.90201 12.313 1.90201 11.68 2.29301 11.29C2.68401 10.899 3.31601 10.899 3.70701 11.29L8.99901 16.586L20.291 5.29301C20.682 4.90201 21.314 4.90201 21.705 5.29301C22.096 5.68401 22.096 6.31601 21.705 6.70701L9.70601 18.707C9.51901 18.895 9.26501 19 8.99901 19" stroke="currentColor"/>
+</svg>
```

Comment:
Why are we not reusing the existing confirm icon?
<img width="228" alt="Screenshot 2023-01-25 at 13 04 02" src="https://user-images.githubusercontent.com/381895/214558689-433f884f-5fb0-40bb-834c-68709bec80c3.png">

Reply:
We are pushing the use of our Icon Library icons. I've already given the same response in other PR so I suggest if we are not completely aligned in this regard that we briefly talk about it after tomorrow's standup.

The next iteration would be to replace other uses of MUI icons as in the case of `TxSummary`

Reply:
It's not what I asked. I got the memo that we're pushing our own icon library, that's fine. I'm asking why the two icons that are supposed to be the same are using a different svg.

Reply:
In other words, I suggest we replace icons in a more systematic way. E.g. use the existing checkmark in this PR and update all of them in a separate PR. Otherwise we'll have a mix of both sprinkled around the app.

Reply:
Alright. I agree with this. Will use the MUI icon in this PR.